src/dataIntegration/transcriptomics/EFlux/eFlux.m

```diff
@@ -0,0 +1,142 @@
+function [foldChange,standardError,solControl,solCondition] = eFlux(model,controlExpression,conditionExpression,varargin)
```

Comment:
May you name the subfolder `eFlux` and add more comments in the different functions?